src/PlatformHelper.cs

```diff
@@ -450,6 +450,17 @@ internal static DateTimeOffset ConvertStringToDateTimeOffset(string text)
         }
 
         /// <summary>
+        /// Converts a string to a DateTimeOffset.

```

Comment:
This file is included by all libraries, like ODataLib, EdmLib, Spatial, Client. So please update the code change to make sure it is only for client.

Please add

#if ASTORIA_CLIENT
...
#endif